Update dumpling-overview.md (#16620)
pingcap/docs: dumpling-overview.md

| `--csv-null-value`           | Representation of null values in CSV files                                                                                                                                                                                                                                                                                         | "\\N"                                      |
| `--csv-line-terminator`      | The terminator at the end of a line for CSV files. When exporting data to a CSV file, you can specify the desired terminator with this option. This option supports "\\r\\n" and "\\n". The default value is "\\r\\n", which is consistent with the earlier versions. Because quotes in bash have different escaping rules, if you want to specify LF (linefeed) as a terminator, you can use a syntax similar to `--csv-line-terminator $'\n'`. | "\\r\\n" |

> | `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to 'redshift'. Note that the '0x' prefix will be removed. For example, '0x61' is represented as '61'.  | ""  |

Reviewer: ```suggestion
| `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to `"redshift"`. Note that the `0x` prefix will be removed. For example, `0x61` is represented as `61`.  | ""  |
```

> | `--csv-output-dialect`      | The value can be 'snowflake', 'redshift', or 'bigquery', indicating that the source data can be exported to CSV in the format required by the aforementioned databases. Such as binary data types will be converted to hexadecimal, but the '0x' prefix will be omitted. For example, '0x61' will be represented as '61'  | ""  |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. For example, binary data types will be converted to hexadecimal, but the '0x' prefix will be removed. For example, '0x61' will be represented as '61'  | ""  |
```

> | `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | ""  |
```

> | `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. For example, if you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64.  | ""  |
```

> | `--csv-output-dialect`          | Dumpling will represent binary data types with UTF-8 format in CSV files by default. To export binary data in another format like HEX, set the parameter to 'redshift', but the '0x' prefix will be omitted, such as '0x61' being represented as '61'.                                                                                                                                                                                                                                                                  | ""                                      |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Dumpling represents binary data types in CSV files in UTF-8 format by default. To export binary data in another format such as HEX, set the parameter to 'redshift'. Note that the '0x' prefix will be removed. For example, '0x61' is represented as '61'.  | ""  |
```

> | `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | ""  |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | `""`  |
```

> | `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string will be garbled. | `""`  |

Reviewer 2: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string might contain garbled characters. | `""`  |
```

> | `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `"snowflake"`, `"redshift"`, and `"bigquery"`. If you set the option to `"snowflake"` or `"redshift"`, the binary data type will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. The default value is null, which means to encode and export the data according to UTF-8. In some cases, the binary string might contain garbled characters. | `""`  |

Reviewer: ```suggestion
| `--csv-output-dialect`      | Indicates that the source data can be exported to a CSV file in a specific required format for the database. The option value can be `""`, `"snowflake"`, `"redshift"`, or `"bigquery"`.  The default value is `""`, which means to encode and export the source data according to UTF-8. If you set the option to `"snowflake"` or `"redshift"`, the binary data type in the source data will be converted to hexadecimal, but the `0x` prefix will be removed. For example, `0x61` will be represented as `61`. If you set the option to `"bigquery"`, the binary data type will be encoded using base64. In some cases, the binary strings might contain garbled characters. | `""`  |
```
